code-challenges.test.js

```diff
@@ -23,37 +23,132 @@ const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// describe('capIt', () => {
+//     it("Takes in an array of words and returns an array with all the words capitalized", () => {
+//         const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+//         // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+//         const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+//         // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+//         expect(capIt(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         expect(capIt(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+//     })
+// })
+
+
+// ReferenceError: capIt is not defined
 
 // b) Create the function that makes the test pass.
 
+// Create a function named capIt that takes in the array and capitalize each strings first character.
+// This function will utilize methods such as .map(), .toUpperCase, and  newly found .substring (used so that the rest of the element does not get capitalized)
+// Return will bring back a new array with the index 0 capitalized only.
+
+// const capIt = (array) => {
+//     return array.map((value) => {
+//         return value[0].toUpperCase() + value.substring(1)
+//     })
+// }
+
+// Test Suites: 1 passed, 1 total
+
+
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const vowelTester1 = "learn"
-// Expected output: 1
-const vowelTester2 = "academy"
-// Expected output: 0
-const vowelTester3 = "challenges"
-// Expected output: 2
+
+
+describe('firstVowel', () => {
+    it('Takes in a string and logs the index of the first vowel', () => {
+        expect(firstVowel(vowelTester1)).toEqual(1)
+        expect(firstVowel(vowelTester2)).toEqual(0)
+        expect(firstVowel(vowelTester3)).toEqual(2)
+        const vowelTester1 = "learn"
+        // Expected output: 1
+        const vowelTester2 = "academy"
+        // Expected output: 0
+        const vowelTester3 = "challenges"
+        // Expected output: 2
+    })
+})
+
+// ReferenceError: firstVowelIndex is not defined
 
 // b) Create the function that makes the test pass.
+
+// create a function named firstVowel
+// The purpose of the function will be to take in strings and return it with the first vowel capitalized 
+// utilize .split and .length on the array 
+// Iterate through using for loops to determine what are vowels
+// conditional statments utilizing && to chain, 
+// return using .index
+// if condtions are met the array will return  with a capitalized vowel
+
+
+const firstVowel = (str) => {
+    let strArr = str.split("")
```

Comment:
For loops can iterate on both strings and arrays since both are indexed.  Overall, great work in this section!